R/adjust.R

```diff
@@ -82,6 +83,12 @@ adjust_infection_to_report <- function(infections, delay_defs,
                                        reporting_model, reporting_effect,
                                        type = "sample",
                                        truncate_future = TRUE) {
+  #nolint start: missing_argument_linter
```

Comment:
why nolint here? Don't see the need to have a missing argument.

Reply:
That's a good point. I initially thought it was because `lintr` thought I was calling a function without arguments but turns out it was the trailing comma. I've removed the trailing comma and removed the tags in 42c9f84df7d08c74354f8781bfd7e909d55e57ab and 5dfe07d065cbd3e8de0d9fe3a5194847a1261609.